main.js

```diff
@@ -0,0 +1,203 @@
+// Grab HTML elements
+const fromTower = document.querySelector(".fromTower");
+const toTower = document.querySelector(".toTower");
+const moveButton = document.querySelector("#moveButton");
+const reset = document.querySelector("#reset");
+
+const tower1 = document.querySelector(".tower1");
+const tower2 = document.querySelector(".tower2");
+const tower3 = document.querySelector(".tower3");

```

Comment:
there are a lot of hard coded values that would prevent the code from being reusable or implement a new peg for example